### PR TITLE
Add MTE-2224 [V124] custom URL new tab test

### PR DESCRIPTION
**Pull request by dragosb01**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-2224)

## :bulb: Description
https://testrail.stage.mozaws.net/index.php?/cases/view/2306875
Screenshot 2024-01-30 at 17 24 03

### CODE REVIEW

File: firefox-ios/Client/Application/ImageIdentifiers.swift [hunk 1]
```diff
@@ -73,5 +73,4 @@ public struct ImageIdentifiers {
     public static let zoomIn = "menu-ZoomIn"
     public static let customizeHomepage = "editLarge"
     public static let help = "helpCircleLarge"
-    public static let share = "Share"
```
Comment (isabelrios) [inline firefox-ios/Client/Application/ImageIdentifiers.swift:76]: Is this intentional? wondering if there might be a test affected?

Comment (dragosb01) [reply firefox-ios/Client/Application/ImageIdentifiers.swift:76]: > Is this intentional? wondering if there might be a test affected?

This is intentional. It is an overdue update based on @thatswinnie request from this PR: https://github.com/mozilla-mobile/firefox-ios/pull/18301

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/NewTabSettings.swift [hunk 2]
```diff
@@ -129,6 +129,36 @@ class NewTabSettingsTest: BaseTestCase {
         validateKeyboardIsRaisedAndDismissed()
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2306875
+    // Smoketest
+    func testNewTabCustomURLKeyboardNotRaised() {
```
Comment (isabelrios) [inline firefox-ios/firefox-ios-tests/Tests/XCUITests/NewTabSettings.swift:134]: Looks like it failes 3 out of 3 in Bitrise :(

Comment (dragosb01) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/NewTabSettings.swift:134]: I am looking into it, there is a bug locally, investigation in progress.

**Review by isabelrios — APPROVED** [review]

**Pull request merged**